pkg/Library/CoreXenon/Framework/Flan.js

```diff
@@ -78,21 +77,34 @@ export const forwardBoundInput = ({flan, onvalue}, scopedInput) => {
   }
 };
 
-export const forwardStateChanges = (flan, inputs) => {
+export const forwardStateChanges = (flan, inputs, justTheseNodes) => {
   values(flan.layers).forEach(layer => {
     const boundInput = Binder.mapInputToBindings(inputs, layer.bindings);
-    const inputsByAtom = create(null);
+    let inputsByAtom = create(null);
     map(boundInput, (id, value) => {
       const atomId = Id.sliceId(id, 0, -1);
       const propId = Id.sliceId(id, -1);
       const inputs = (inputsByAtom[atomId] ??= create(null));
       inputs[propId] = value;
     });
+    if (justTheseNodes) {
+      inputsByAtom = filterAtomMapByNodeIds(inputsByAtom, justTheseNodes);
+    }
     const {atoms} = layer;
     map(inputsByAtom, (id, inputs) => atoms[id] && (atoms[id].inputs = inputs));
   }); 
 };
 
+const filterAtomMapByNodeIds = (byAtom, justTheseNodes) => {
+  const filtered = create(null);
+  justTheseNodes.forEach(nodeId => map(byAtom, (id, inputs) => {
+    if (id.startsWith(nodeId)) {
```

Comment:
do we need to make a `nodePrefix = ${nodeId}$`,
so that `Echo2` is not matched, when `justTheseNodes` contains `Echo`?

Reply:
Good catch! We should have an `Id` function to test prefixes properly.

Also, fwiw,  `justTheseNodes` entries must be of the form `<layer>$<nodeId>`.